docker-compose-installed.yaml

```diff
@@ -115,15 +123,20 @@ services:
       context: .
       dockerfile: ./rpc/Dockerfile-installed
       args:
-        ISOLATION_ID: ${ISOLATION_ID}
+        - ISOLATION_ID=${ISOLATION_ID}
+        - http_proxy
+        - https_proxy
+        - no_proxy
     image: sawtooth-seth-rpc:${ISOLATION_ID}
     container_name: seth-rpc
     volumes:
       - ./build/debs:/build
     depends_on:
       - validator
+    expose:
```

Comment:
The `ports` option publishes port 3030 to the host machine, which we need in order to send requests from the host machine. `expose` will make port 3030 available to other containers in the docker network, but this is already implied by the `ports` option.

Reply:
Thanks. I will remove expose and test with ports only.